templates/deployment.yaml

```diff
@@ -15,14 +15,8 @@ spec:
     metadata:
       annotations:
         checksum/config: {{ include (print $.Template.BasePath "/config.yaml") . | sha256sum }}
```

Comment:
I believe, but just confirming, that this change + latest-version is going out as a 3.x branch?

Reply:
Well I am merging them into main. I made a 2.x branch for support of the other release branch, same as I did for the 1.x -> 2.x transition.

Reply:
Ah, sorry. I meant to say "going out as a 3.0 release".